src/overview/structure.md

```diff
@@ -18,19 +26,25 @@ yourproject/
 
-There is always exactly one Router per cluster.  The Router of a cluster is a single nginx process.  It is configured by the `routes.yaml` file.  Its purpose is to map incoming requests to an Application container, and to provide basic caching of responses if so configured. It has no persistent storage.
+There is always exactly one Router per cluster.
+
+The Router of a cluster is a single nginx process.  It is configured by the `routes.yaml` file.  It maps incoming requests to the appropriate Application container and provides basic caching of responses, if so configured. It has no persistent storage.
 
 ## Service
 
-Service containers are configured by the `services.yaml` file.  There may be zero or more service containers in a cluster, depending on the `services.yaml` file.  The code for a Service is provided by Platform.sh in a pre-built container image, along with default configuration.  Depending on the service it may also have user-provided configuration in the `services.yaml` file.  Examples of services include MySQL/MariaDB, Elasticsearch, Redis, and RabbitMQ.
+Service containers are configured by the `services.yaml` file.
+
+There may be zero or more Service containers in a cluster, depending on the `services.yaml` file.  The code for a Service is provided by Platform.sh in a pre-built container image, along with a default configuration.  Depending on the service it may also include user-provided configuration in the `services.yaml` file.  Examples of services include MySQL/MariaDB, Elasticsearch, Redis, and RabbitMQ.
 
 ## Application
 
-There is always at least one Application container per cluster, but there may be more.  Each application container corresponds to a `.platform.app.yaml` file in the repository.  If there are 3 `.platform.app.yaml` files, there will be three Application containers.  Application containers hold the code you provide via your Git repository.  Application containers are always built off of one of the Platform.sh-provided language-specific images, such as “PHP 5.6”, “PHP 7.2”, or “Python 3.6”. And different application containers can be running different languages or versions.
+There always must be one Application container in a cluster, but there may be more.
+
+Each Application container corresponds to a `.platform.app.yaml` file in the repository.  If there are 3 `.platform.app.yaml` files, there will be three Application containers.  Application containers hold the code you provide via your Git repository.  Application containers are always built off of one of the Platform.sh-provided language-specific images, such as “PHP 5.6”, “PHP 7.2”, or “Python 3.6”. It is also possible to have multiple Application containers running different languages or versions.
 
-For most typical applications there is only one `.platform.app.yaml` file, which is generally placed at the repository root.
+For many applications, there is only one `.platform.app.yaml` file, which is generally placed at the repository root.
```

Comment:
Why this change?  Typical projects are single-app, so this is true.

Reply:
Fixed.